aspnetcore/migration/21-to-22.md

```diff
@@ -107,6 +107,26 @@ public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
         });
 ```
 
+If the app doesn't call `CreateDefaultBuilder`, call <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel*> **before** calling `ConfigureKestrel`:
```

Comment:
I don't think you need a code sample. I think just saying:

If the app doesn't call `CreateDefaultBuilder`, UseKestrel <xref:Microsoft.AspNetCore.Hosting.WebHostBuilderKestrelExtensions.UseKestrel*> still needs to be called, and should be **before** calling `ConfigureKestrel`.

Reply:
> don't think

I ~picture~ *code sample* is worth a thousand words. :smile:

I'll strike it, of course, if you like. There's no harm tho.

Reply:
The current text says what you just said without colloquial language ("still") and avoids the state verb ("be").

Reply:
Haha okay keep it 😄 .